package.json

```diff
@@ -53,5 +52,8 @@
     "webpack": "4.27.1",
     "webpack-bundle-tracker": "0.3.0",
     "webpack-cli": "3.1.2"
+  },
+  "resolutions": {
+    "deep-extend": ">=0.5.1"
   }
```

Comment:
What is this? I've never used `resolutions` before.

Reply:
This is [Yarn's "selective dependency resolution" feature](https://yarnpkg.com/en/docs/selective-version-resolutions#search). It allows us to influence the algorithm it uses to resolve dependency versions. It's sort of like a Python constraints file in that regard. It's one of the neat things about Yarn that I don't know how to reproduce in NPM.

We had resolved `deep-extend@0.4.2` for one of our dependencies. That version had a minor security problem that `yarn audit` detected. This line forces Yarn to only install versions `>=0.5.1`, where the security problem was fixed (in practice it resolved it 0.6.0). You can see this change [in the lock file as well](https://github.com/mozilla/normandy/pull/1645/files#diff-8ee2343978836a779dc9f8d6b794c3b2L2080).